src/EditorFeatures/CSharpTest/Diagnostics/UseImplicitOrExplicitType/UseExplicitTypeTests.cs

```diff
@@ -1186,5 +1186,31 @@ static void M()
 }",
 options: ExplicitTypeEverywhere());
         }
+
+        [WpfFact, Trait(Traits.Feature, Traits.Features.CodeActionsUseExplicitType)]
```

Comment:
Off the top of my head, no idea. Switch it to Fact locally and see what happens!